pom.xml

```diff
@@ -185,21 +181,21 @@
       <artifactId>maven-dependency-tree</artifactId>
       <version>3.2.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
```

Comment:
? really needed? this doesn't look related to the subject of the PR

Reply:
? unsure I get your comment. If you'd look carefully, you will see that I just moved this dependency from last place (as it is non test) to here. I just "moved" it to have proper dep order.

Reply:
ah right. sorry for that.
aren't we using spotless for pom?